components/activeLink.js

```diff
@@ -0,0 +1,51 @@
+import { useRouter } from "next/router";
+
+const ActiveLink = () => {
+  const handleClick = (e) => {
+    const router = useRouter();
+    e.preventDefault();
+    router.push(href);
+  };
+  return (
+    <>
```

Comment:
This should be pulled out and turned into a Nav component. The Nave will then import the activeLink component for the buttons. As for this component, you should follow the example here almost verbatim. https://nextjs.org/docs/api-reference/next/router